model/dhcp-agent.yang

```diff
@@ -0,0 +1,360 @@
+module dhcp-agent {
+
+    namespace "urn:ietf:params:xml:ns:yang:dhcp-agent";
+    prefix dhcp-agent;
+
+    import ietf-inet-types {
```

Comment:
Specify the exact revision of modules to avoid problems with tools which apply different interpretations to RFC 6020/7950 rules.

Reply:
YANG issues are open issues for wider discussion.